examples/google-signin/google-signin-iframe.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <script async src="https://news.google.com/swg/js/v1/google_signin.js"></script>
+  <script src="<% swgGsiJsUrl %>"></script>
```

Comment:
I like the dynamic changing but I worry that it will cause confusion for publishers. This would be okay to do if we have a comment describing how to inject this URL and where the production one lives.

Reply:
I added a comment above the script